packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -20,6 +20,9 @@ DevTools extension for your pub package, see the getting started guide for
 
 * Fixed theming bug in isolate selector - [#6403](https://github.com/flutter/devtools/pull/6403)
 
+* Adds `requestAndroidAppLinkSettings`, `requestIosBuildOptions`, and
+  `requestIosUniversalLinkSettings` endpoints to ServerApi.
+
```

Comment:
The release notes are intended for user-facing changes, so let's hold off on adding this note here until we ship the deep link feature in general. The entry you added to `devtools_shared/CHANGELOG.md` is sufficient. To get the release notes check on the bots, add the label "release notes not required"